docs/configuration/install.md

```diff
@@ -214,11 +214,16 @@ systemctl restart httpd.service
 <a name="php-installationconfiguration"></a>
 ## PHP Installation/Configuration
 
-On Debian/Ubuntu execute: 
+On Debian/Ubuntu 14.04 execute:
 ```
 apt-get install libapache2-mod-php5
 ```
 
+On Ubuntu 16.04 execute:
+```
+apt-get install libapache2-mod-php7
```

Comment:
Should we consider to use the actual commands? In this case it would be `apt-get install libapache2-mod-php7.0`. There are also versions for PHP 7.1 and 7.2.

IMO the Chapter `Install other Dependencies` is good example for this. The commands can be copy and pasted in the cli.
What do you think?

Reply:
I meant `apt-get install libapache2-mod-php7.0`. I corrected it.